lib/diaspora_federation/discovery/web_finger.rb

```diff
@@ -1,7 +1,7 @@
 module DiasporaFederation
   module Discovery
-    # The WebFinger document used for Diaspora* user discovery is based on an older
-    # draft of the specification you can find in the wiki of the "webfinger" project
+    # The WebFinger document used for diaspora* user discovery is based on an older
+    # draft of the specification which you can find in the wiki of the "webfinger" project
     # on {http://code.google.com/p/webfinger/wiki/WebFingerProtocol Google Code}
```

Comment:
This link doesn't work anymore. but I found https://tools.ietf.org/html/draft-jones-appsawg-webfinger-06